lib/archives_sidebar/lib/archives_sidebar.rb

```diff
@@ -30,7 +30,7 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i % 12) + 1
```

Comment:
Good catch! Given that the rubo-cop errors are in a different area of code, and not related to your changes, please pull them out. If you'd like to create a new issue/pr for the linting issue, we would welcome it!